ui/components/multichain/edit-accounts-modal/edit-accounts-modal.test.tsx

```diff
@@ -40,10 +41,21 @@ const render = (
     mockState.metamask.internalAccounts.accounts,
   ) as unknown as MergedInternalAccount[];
 
+  const accountsWithCaipAccountId = accounts.map((account) => {
+    const { namespace, reference } = parseCaipChainId(account.scopes[0]);
```

Comment:
Why do we need to parse here? I would assume that the scope is always valid and we can prefix without parsing

Reply:
good point! https://github.com/MetaMask/metamask-extension/pull/29887/commits/8bea6c8002dc6258a75c65247f65fb86e9fec742